plugin/src/steps/check-gatsby-version.js

```diff
@@ -0,0 +1,23 @@
+const semver = require(`semver`)
+const fs = require(`fs`)
+
+export function checkGatsbyVersion({ reporter }) {
```

Comment:
@TylerBarnes I think this is how we'd do the version check we discussed, yeah?

Reply:
This looks good to me 👍 I think you mean minimum instead of minimim though :)

Reply:
LOL

Reply:
Let me know if you want any other tweaks to that message

Reply:
It might be helpful for users if we mention why it's a requirement. Something like `The minimum supported Gatsby version is 2.24.74. This is because previous versions do not support the new "pluginOptionsSchema" node API which this plugin is now implementing.` or something like that